src/index.ts

```diff
@@ -507,10 +636,65 @@ export function getOptions(options: RokuDeployOptions = {}) {
     return finalOptions;
 }
 
+export function getStagingFolderPath(options?: RokuDeployOptions): string {
```

Comment:
Remove the `: string` type here, it is inferred by typescript and is therefore not needed.